README.md

```diff
@@ -71,7 +71,8 @@ See [Docker installation instructions](DOCKER.md).
 
 1. Run `git clone git@github.com:orientation/orientation.git` in Terminal.
 2. `cd` into the cloned directory.
-3. Run `rake orientation:install` in Terminal. This will install gem dependencies.
+3. Run `bin/setup` in Terminal. This will install project dependencies.
+  - Run `USE_OSX_SERVICES=true bin/setup` if you don't plan on using Docker to run system dependencies
```

Comment:
Unfortunately, `rake` won't work if you've just installed a new version of Ruby. Using `bin/setup` allows for a more agnostic way of setting up the appropriate dependencies.

Reply:
Huh.